lib/pages/chat/send_file_dialog/send_file_dialog_view.dart

```diff
@@ -27,7 +22,9 @@ class SendFileDialogView extends StatelessWidget {
         borderRadius:
             BorderRadius.circular(SendFileDialogStyle.dialogBorderRadius),
         child: Container(
-          width: SendFileDialogStyle.dialogWidth,
+          width: controller.isSendMediaWithCaption
+              ? SendFileDialogStyle.dialogWidthForMedia
+              : SendFileDialogStyle.maxDialogWidth,
```

Comment:
why need 2 kinds of width? why not use the same?

Reply:
i based on the design, and when i use the same width for media, it's not look good for one image for now